bcgov/biohubbc
# SIMSBIOHUB-442: Add support for hiding/showing table columns for Obserations and Telemetry

## Links to Jira Tickets

 - https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-442

## Description of Changes

- Add support for hiding/showing table columns for Obserations and Telemetry
- Added `sessionStorage` call in order to cache column visibility across app reloads
- Renamed `ObservationComponent` and `ManualTelemetryComponent` to `ObservationsTableContainer` and `ManualTelemetryTableContainer`

## Testing Notes

- Visit the Manage Observations or Manage Telemetry page
- Click the cog icon
- You should be able to:
  - [ ] Show/hide all columns
  - [ ] Show/hide additional columns
 - Show/hide some or all columns, then refresh the page. You should:
   - [ ] See that the visiblity state has persisted across refreshes
